dhall/tests/format/multilineTrailingWhitespaceB.dhall

```diff
@@ -1,6 +1,6 @@
 { x =
     ''
-      
+
```

Comment:
I wasn't aware that leading whitespace would be ignored by the parser when I originally added this testcase.